src/add-item.js

```diff
@@ -58,7 +74,15 @@ async function getItemConfig(configs) {
           type: 'input',
           name: 'reportedDate',
           message: 'Enter a xlsform calculation formular to calculate the reported date',
-          default: 'now()'
+          default: 'now()',
+          when: function(answers){
```

Comment:
good stuff here 👍.
Maybe I'm missing something here as well, I'm trying to get the usecase of these `when`  blocks

Reply:
ah okay i see
Good